lib/cable_ready/channels.rb

```diff
@@ -4,12 +4,27 @@
 
 module CableReady
   class Channels
+    include Singleton
+    attr_accessor :operations
+
+    def self.configure
+      yield CableReady::Channels.instance if block_given?
+    end
+
     def initialize
       @channels = {}
+      @operations = {}
+      [:add_css_class, :dispatch_event, :inner_html, :insert_adjacent_html, :insert_adjacent_text, :morph, :outer_html, :remove, :remove_attribute, :remove_css_class, :set_attribute, :set_cookie, :set_dataset_property, :set_property, :set_style, :set_styles, :set_value].each do |operation|
```

Comment:
Have you considered using the literal notation for an array of symbols? It would avoid `,` and `:` and make the code a little bit easier to read.

Reply:
Do you mean %w[ this burns my eyes ]

I hadn't considered it, because that generates an array of strings, not Symbols.

Honestly, I have never loved %w[] and it could just be subjective, but I feel like an array of Symbols is more idiomatic in the wild. What do you think, @hopsoft?

Reply:
Here's my preferred syntax/formatting for lists like this.

```ruby
operation_names = %i[
  add_css_class
  dispatch_event
  inner_html
  insert_adjacent_html
  insert_adjacent_text
  morph
  outer_html
  remove
  remove_attribute
  remove_css_class
  set_attribute
  set_cookie
  set_dataset_property
  set_property
  set_style
  set_styles
  set_value
]
```